app/components/sidebars/main/channels_list/list/list.js

```diff
@@ -87,67 +81,157 @@ export default class List extends PureComponent {
         }
     }
 
-    buildSections = (props) => {
-        const {
-            canCreatePrivateChannels,
-            directChannelIds,
-            favoriteChannelIds,
-            publicChannelIds,
-            privateChannelIds,
-            unreadChannelIds,
-        } = props;
-        const sections = [];
+    getSectionConfigByType = (sectionType) => {
+        const {canCreatePrivateChannels} = this.props;
 
-        if (unreadChannelIds.length) {
-            sections.push({
+        switch (sectionType) {
+        case SidebarSectionTypes.UNREADS:
+            return {
                 id: 'mobile.channel_list.unreads',
                 defaultMessage: 'UNREADS',
-                data: unreadChannelIds,
                 renderItem: this.renderUnreadItem,
-                topSeparator: false,
-                bottomSeparator: true,
-            });
-        }
-
-        if (favoriteChannelIds.length) {
-            sections.push({
+            };
+        case SidebarSectionTypes.FAVORITE:
+            return {
                 id: 'sidebar.favorite',
                 defaultMessage: 'FAVORITES',
-                data: favoriteChannelIds,
-                topSeparator: unreadChannelIds.length > 0,
-                bottomSeparator: true,
-            });
+            };
+        case SidebarSectionTypes.PUBLIC:
+            return {
+                action: this.goToMoreChannels,
+                id: 'sidebar.channels',
+                defaultMessage: 'PUBLIC CHANNELS',
+            };
+        case SidebarSectionTypes.PRIVATE:
+            return {
+                action: canCreatePrivateChannels ? this.goToCreatePrivateChannel : null,
+                id: 'sidebar.pg',
+                defaultMessage: 'PRIVATE CHANNELS',
+            };
+        case SidebarSectionTypes.DIRECT:
+            return {
+                action: this.goToDirectMessages,
+                id: 'sidebar.direct',
+                defaultMessage: 'DIRECT MESSAGES',
+            };
+        case SidebarSectionTypes.RECENT_ACTIVITY:
+            return {
+                action: this.goToDirectMessages,
+                id: 'sidebar.types.recent',
+                defaultMessage: 'RECENT ACTIVITY',
+            };
+        case SidebarSectionTypes.ALPHA:
```

Comment:
It's used when the channel grouping is set to never and the sort is set to alphabetical.

Reply:
Oh! So this is only set in the webapp then. I hadn't been able to find it, since it looks like it's currently hard-coded in that branch as just a string vs. a const. Thanks :)